Sources/ProjectDescription/TargetAction.swift

```diff
@@ -57,6 +60,7 @@ public struct TargetAction: Codable, Equatable {
         case outputPaths
         case outputFileListPaths
         case basedOnDependencyAnalysis
+        case runForInstallBuildsOnly = "runOnlyForDeploymentPostprocessing"
```

Comment:
is there a reason why the raw value is different from the case name?

Reply:
was not sure if `TargetAction` get's persisted. If not maybe you remove it and also the miss-leading `Codable` ?

Reply:
was thinking the persisted value has to match `PBXShellScriptBuildPhase`?

Reply:
all models in `ProjectDescription` should be codable but that's only used to parse the manifests. Then they are converted to their internal representations defined in `TuistGraph`, so nothing in this module should affect anything in `XcodeProj` layer